lib/dolly/document.rb

```diff
@@ -143,9 +143,16 @@ def self.property *ary
       options           = ary.pop if ary.last.kind_of? Hash
       options         ||= {}
 
-      ary.each do |name|
+      if ary.count==1 && options[:class_name] == Hash && block_given?
```

Comment:
in retro spec i dont think i need to limit it to 1 property per block, although each property would have all the sub properties under it, unless perhaps i give it an `:only` option, kind of like in rails routing.